README.md

```diff
@@ -21,3 +21,10 @@ npm run create docs/user/test/test.md
 ```
 
 Make sure you have draft set to false in the page metadata, otherwise the page you created won't be displayed.
+
+### Page weight
+
+The weight indicates which page comes first in the sidebar. We use this system: SBXX where:
+- S: corresponds to the sidenav section number (begins at 1)
+- B: corresponds to the branch number (set to 0 if not in a branch)
+- XX: place inside the current group of pages (begins at 1)
```

Comment:
could do X0, so you can put a page in between without having to review all weights. For example if you have to pages weighted `2010` and `2020` it's easier to add a page between them with the weight `2011`.

Reply:
It should be SBXX0 then, as there can be more than 9 pages in a section